package.json

```diff
@@ -12,11 +12,11 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "db:migrate": "sequelize db:migrate",
     "db:migrate-undo": "sequelize db:migrate:undo:all",
-    "db:rollback": "npm run db:back && npm run db:roll && npm run db:seed",
+    "db:rollback": "npm run db:migrate-undo && npm run db:migrate && npm run db:seed",
```

Comment:
ditto

Reply:
the db migrate script was formerly `db:roll` and you asked it to be changed in to `db: migrate` but the change wasn't reflected in the `db:rollback` script before merging. so for the script to work, I had to change it.